File: SubtractNumbers.cpp
#include <iostream>
using namespace std;

int main() {
    int num1, num2;
    
    cout << "Enter first number: ";
    cin >> num1;
    
    cout << "Enter second number: ";
    cin >> num2;
    
    if (num2 <= num1) {
        int result = num1 - num2;
        cout << "Result: " << result << endl;
    } else {
        cout << "Error: Second number must be smaller than or equal to the first number." << endl;
    }
    
    return 0;
}

File: SubtractNumbers.py
num1 = int(input("Enter first number: "))
num2 = int(input("Enter second number: "))

if num2 <= num1:
    result = num1 - num2
    print("Result:", result)
else:
    print("Error: Second number must be smaller than or equal to the first number.")
